### PR TITLE
Add support for exporting MulConstant, DivConstant and Softmax to ONNX

### PR DESCRIPTION
**Pull request by bddppq**

### CODE REVIEW

File: torch/autograd/_functions/basic_ops.py [hunk 1]
```diff
@@ -169,6 +169,14 @@ def backward(ctx, grad_output):
 
 @traceable
 class MulConstant(InplaceFunction):
+    @staticmethod
+    def symbolic(g, a, b, inplace=False):
+        assert not inplace, "Exporting inplace MulConstant is not supported yet"
```
Comment (ezyang) [inline torch/autograd/_functions/basic_ops.py]: We are never supporting inplace MulConstant, so this line should just be deleted.

File: torch/autograd/_functions/basic_ops.py [hunk 2]
```diff
@@ -169,6 +169,14 @@ def backward(ctx, grad_output):
...
+        if isinstance(a, torch._C.Node):
```
Comment (ezyang) [inline torch/autograd/_functions/basic_ops.py]: Can we write a helper function here similar to `sort_args`; most importantly to avoid duplicating the very ugly `isinstance` check.

**Comment by ezyang** [comment]
Thanks. This also partially fixes https://github.com/pytorch/pytorch/issues/2747
